edweiss-app/components/lectures/remotecontrol/modal.tsx

```diff
@@ -148,3 +151,204 @@ const TwoLangsSelection: React.FC<TwoLangsSelectionProps & LightDarkProps> = ({
+    }
+
+    const ppMinute = (seconds: number) => {
+        const minutes = Math.floor(seconds / 60);
+        return `0${minutes % 60}`.slice(-2);
+    }
+
+    const ppHour = (seconds: number) => {
+        return `0${Math.floor(seconds / 3600)}`.slice(-1);
+    }
+
+    // Timer Seter interface 
+    const setTimerInterf = (seconds: number) => {
+        if (0 <= seconds && seconds <= 35999) {
+            setTmpTimer(seconds);
+        }
+    }
+
+    // Recall Seter interface
+    const setRecallInterf = (seconds: number) => {
+        if (0 <= seconds && seconds <= 35999) {
+            setTmpRecall(seconds);
+        }
+    }
+
+    return (
+        <ModalContainer modalRef={modalRef}>
+            <>
+                <TView justifyContent='center' alignItems='center' mb='sm'>
+                    <TText bold size='lg' mb='sm'>{t('showtime:rmt_ctl_set_timer_title')}</TText>
+                </TView>
+
+
+
+                <TView mt={'sm'}>
+
+                    {/* Timer Display */}
+                    <TView flexDirection='row' justifyContent='space-between' mb={'md'} >
+                        <TView flexDirection='row' justifyContent='space-between' flex={1} mt={'xs'}>
+
+                            <TText ml={'md'} mt={'md'} size={'lg'}>{t('showtime:rmt_ctl_timer')}</TText>
+
+                            {/* Timer Hours */}
+                            <TTouchableOpacity ml={'lg'} radius={'md'} borderColor={butSelcted === 0 ? 'blue' : 'surface2'} b={1} justifyContent='center' pr={'md'} pl={'md'} onPress={() => { setButSelcted(0); setCurCoef(hoursCoef); }} testID='hours-timer'>
+                                <TText size={'lg'} color={butSelcted === 0 ? 'blue' : 'surface1'} testID='tmp-timer-hours'>{ppHour(tmpTimer)}</TText>
+                            </TTouchableOpacity>
+
+                            <TText size={'xl'} pt={'lg'}>:</TText>
+
+                            {/* Timer Minutes */}
+                            <TTouchableOpacity radius={'md'} borderColor={butSelcted === 1 ? 'blue' : 'surface2'} b={1} justifyContent='center' pr={'md'} pl={'md'} onPress={() => { setButSelcted(1); setCurCoef(minutesCoef); }} testID='minutes-timer'>
+                                <TText size={'lg'} color={butSelcted === 1 ? 'blue' : 'surface1'} testID='tmp-timer-minutes'>{ppMinute(tmpTimer)}</TText>
+                            </TTouchableOpacity>
+
+                            <TText size={'xl'} pt={'lg'} >:</TText>
+
+                            {/* Timer Seconds */}
+                            <TTouchableOpacity mr={'lg'} radius={'md'} borderColor={butSelcted === 2 ? 'blue' : 'surface1'} b={1} justifyContent='center' pr={'md'} pl={'md'} onPress={() => { setButSelcted(2); setCurCoef(secondsCoef); }} testID='seconds-timer'>
+                                <TText size={'lg'} color={butSelcted === 2 ? 'blue' : 'surface1'} testID='tmp-timer-seconds'>{ppSeconds(tmpTimer)}</TText>
+                            </TTouchableOpacity>
```

Comment:
Maybe you could return back to the line to make the parameters more readable e.g 
```
{/* Timer Hours */}
<TTouchableOpacity 
     radius={'md'} 
     borderColor={butSelcted === 0 ? 'blue' : 'surface2'} 
     justifyContent='center' 
     b={1} pr={'md'} pl={'md'} ml={'lg'} 
     onPress={() => { setButSelcted(0); setCurCoef(hoursCoef); }} 
     testID='hours-timer'
>
     <TText size={'lg'} color={butSelcted === 0 ? 'blue' : 'surface1'} testID='tmp-timer-hours'>                
             {ppHour(tmpTimer)}
     </TText>
</TTouchableOpacity>

Reply:
Of course! If you prefer it this sure, let's give it a try.